lib/org/chromium/webidl/PipelineMessage.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Any preference on how to clearly identify the input and output messages?

Previously (before using this `PipelineMessage` class), I created an object that only contained items that were needed for the upcoming stages. That object is passed to the `output` method of Box Runnables, which forwards it to the next runner. For example, `this.output({ file: file, parser: parser });` 

However, as the pipeline grew, it started to get messy and confusing as the output of each runner differed every so slightly at every stage.

On a side note, where should items go once they are no longer needed in the pipeline? For example, where should `HTMLFileContents` go after it has been processed in the pipeline? I know we intend to store it in DataStore so we have a trail of the process. However, do we intend on storing the items in DataStore immediately once they are no longer needed in the pipeline?